docs/style_guide.md

```diff
@@ -0,0 +1,28 @@
+# Style Guide
+
+This project follows Google's C++ style guide
+
+https://google.github.io/styleguide/cppguide.html
+
+With some exceptions and clarifiactions listed below
+## Exceptions
+
+### Variable Names
+
+#### Constant Names
+
+Variables declared as constant should be fully capitalized with underscores placed where separation is necessary.
+
+Examples:
+- const byte ROWS = 4;
+- const int WINDOW_SIZE = 10000;
+
+## Clarifications
+
+### Function Names:
```

Comment:
I think the question is what to do with acronyms. I think this explains it. A quote from Google Style Guide: 

> When abbreviations or acronyms appear in such names, prefer to capitalize the abbreviations or acronyms as single words (i.e StartRpc(), not StartRPC()).